devenv/zabbix62/docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
       POSTGRES_DB: zabbix
 
   database:
-    image: postgres
+    image: postgres:15
```

Comment:
there are multiple devenv-docker-compose files with this postgres-entry.. do we want to fix all of them here?

Reply:
yeah I think that makes sense.

Reply:
FYI I had to bump this because postgres translates to the latest which is 16 and zabbix server shuts down with an error:
`Unable to start Zabbix server due to unsupported PostgreSQL database version (16.0). Must not be higher than (15.x)`